### DISCUSSION TITLE
a cmd launcher just for upscallers/postprocessing. (low vram users)

### DISCUSSION THREAD

**Discussion by dudumaroja** [hlky/stable-diffusion-webui#499] [Feature suggestions]
I was for days trying to make SDwebui with upscallers models.. but as a just have 6gb ram (and to make ir worse a 1660 ti, so no half trick for me) everytime i try to use with upscallers evething goes downhill and i cant do nothing..

for now i have two instalations one with upscaling models and one without.. it whould be nice to have a upscallersonly.cmd if it is not too much work, but if noone more cares about this... i could keep the two installations rout.

**Comment by devilismyfriend** [comment]
@hlky maybe we shouldn't load the GFPGAN and RealESRGAN on startup in general and use the model loader/unloader on demand? they load pretty quickly anyway.
